requirements.txt

```
@@ -3,4 +3,4 @@ pytest~=7.4
 pytest-cov~=4.1
 PyYAML~=6.0
 vcrpy~=4.4; python_version == "3.7"
-vcrpy~=5.1; python_version > "3.7"
+vcrpy~=7.0; python_version > "3.7"
```

```suggestion
vcrpy~=5.1; python_version == "3.8"
vcrpy~=7.0; python_version > "3.8"
```

vcrpy 7 is not compatible with Py 3.8.